lib/service/storage/elasticsearch.ts

```diff
@@ -679,14 +742,24 @@ class ElasticSearch extends Service {
    * @returns {Promise.<Object>} { _id, _version, _source }
    */
   async create(
-    index,
-    collection,
-    content,
-    { id, refresh, userId = null, injectKuzzleMeta = true } = {},
+    index: string,
+    collection: string,
+    content: JSONObject,
+    {
+      id,
+      refresh,
+      userId = null,
+      injectKuzzleMeta = true,
+    }: {
+      id?: string;
+      refresh?: boolean | "wait_for";
+      userId?: string;
+      injectKuzzleMeta?: boolean;
+    } = {},
```

Comment:
```suggestion
    {
      id,
      refresh,
      userId = null,
      injectKuzzleMeta = true,
    }: Exclude<KRequestParams, "retryOnConflict" | "timeout"> & { id?: string; } = {},
```

With new `KRequestParams` type we can compose type like it, to DRY